app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -11,7 +11,7 @@
   width: 33%;
   min-height: 150px;
   min-width: 280px;
```

Comment:
Is this line needed? It causes the boxes to wrap onto a second line below around 870px screen width, I can't tell if that's intentional or not?

![screen shot 2018-04-12 at 13 12 37](https://user-images.githubusercontent.com/861310/38676582-621ef3e2-3e53-11e8-958c-356c105da1f6.png)